src/core/customers/interfaces/FindCustomerById.d.ts

```diff
@@ -0,0 +1,5 @@
+import CustomerDTO from "../dto/CustomerDTO";
+
+export default interface FindCustomerByCpf {
+  findByCPF(cpf: string): Promise<CustomerDTO>;
```

Comment:
Oie @vitorrafael tudo bom?
Vi que nos métodos de Order, as buscas estão como GETAlgumacoisa, porém produto também está com a palavra Find.
Acha melhor mudar os métodos de Order para FindAlgumacoisa ou mudamos o de Customer para GetCustomerByCpf?

Reply:
Acho que podemos mudar para Get

Reply:
Cara eu resolvi deixar Get (o que vem da API) de um lado e Find (o que busca no banco) do outro, que esse nome de métodos igual ta fritando meu cérebro kkkk que to me perdendo nas referencias.
Dai pode deixar como esta.